### PR TITLE
Removes the ability to grab anvils with TK.

### PR DESCRIPTION
**Pull request by Sooose**
We removed being able to throw barrels via telekinesis in #28628 and I don't see why we should still have it for anvils. The original intent was for BIG guys with both hulk AND strength to be able to whip these things around, not for scrawny gene nerds to slam them into people through walls without any of the risk (Yes, this is grudge code).

Shoutout to ancientpower and jknpjr for answering my dumb questions.

:cl:
* bugfix: Removed the ability to throw anvils with telekinesis.

### CODE REVIEW

**Comment by Exxion** [comment]
This seems like an unnecessary noob trap. Couldn't you just make it not work?

**Comment by Sooose** [comment]
I don't care either way, my original intent was to just remove the ability to do it but someone suggested I code the brain damage for straining the mind.

**Comment by Kurfursten** [comment]
Exxion is right. Bare minimum the CL should reflect that instead the anvil will injure you.

**Comment by Eneocho** [comment]
> 
> 
> I don't care either way, my original intent was to just remove the ability to do it but someone suggested I code the brain damage for straining the mind.

So the bypass to this PR would be downing a fuckton of alkysine/alkycosine before pulling the anvil?

**Comment by Sooose** [comment]
> > I don't care either way, my original intent was to just remove the ability to do it but someone suggested I code the brain damage for straining the mind.
> 
> So the bypass to this PR would be downing a fuckton of alkysine/alkycosine before pulling the anvil?

You still can't TK the anvil as it was and there was also a small chance of internal bleeding. I'm going to remove both of those and just keeping it at being unable to move the anvil.

**Comment by Jackreturnswithaveangenace** [comment]
Add it back in, it seems funny.

**Comment by Dacendeth** [comment]
Awww all that work to make it an interesting thing down the gutter

File: code/modules/blacksmithing/anvil.dm [hunk 1]
```diff
@@ -42,3 +42,6 @@
 		if(W.loc == src.loc && params)
 			W.setPixelOffsetsFromParams(params, user)
 			return 1
+
+/obj/item/anvil/attack_tk(mob/user)
+	return attack_hand(user)
```
Comment (boy2mantwicethefam) [inline code/modules/blacksmithing/anvil.dm:47]: Won't this remotely grab the anvil and put it in your hand

Comment (Sooose) [reply code/modules/blacksmithing/anvil.dm:47]: Already tested it plenty, It's directly copied from some button code to prevent TK users too.

**Pull request merged**